requirements/edx/base.txt

```diff
@@ -6,6 +6,7 @@
 
 beautifulsoup4==4.1.3
 beautifulsoup==3.2.1
+before_after==0.1.3
```

Comment:
I looked into that, but I wasn't sure it would cover our use case here. If this library is needed for unit tests that will be run on Jenkins, will placing it there still keep things working?